packages/core/src/state-management/repository.ts

```diff
@@ -114,8 +114,8 @@ export class Repository {
     if (commit == null) {
       throw new Error(`No genesis commit found with CID ${genesisCid.toString()}`);
     }
-    // Do not check timestamp here
-    const state = await handler.applyCommit(commit, {cid: streamId.cid, timestamp: null}, this.#deps.context);
+    // Assume here genesis is always earlier than possible key revocation, so do not check a timestamp.
```

Comment:
```suggestion
    // Do not check for possible key revocation here, as we will do so later after loading the tip (or learning that the genesis commit *is* the current tip), when we will have timestamp information for when the genesis commit was anchored. In order to disable this check for key rotation, set a timestamp that will be considered earlier than any other timestamp so it will always appear to have happened earlier than any possible key revocation.
```

Reply:
Tried fleshing out this comment a bit as I think it's tricky and likely to be confusing to future readers without the context, so merits a longer comment.  LMK if you think this is clear?